sbin/umount/umount.c

```diff
@@ -637,7 +661,47 @@ usage(void)
 {
 
 	(void)fprintf(stderr, "%s\n%s\n",
-	    "usage: umount [-fNnv] special ... | node ... | fsid ...",
-	    "       umount -a | -A [-F fstab] [-fnv] [-h host] [-t type]");
+	    "usage: umount [-dfNnv] special ... | node ... | fsid ...",
+	    "       umount -a | -A [-F fstab] [-dfnv] [-h host] [-t type]");
 	exit(1);
 }
+
+int
+md_detach(const char *device)
+{
+	struct md_ioctl mdio;
+	char *p;
```

Comment:
Perhaps it's better to name the variable name `eptr` (or something similar) instead of just `p`.
As that argument is named `endptr`, it's easier to grasp (by that name).

Reply:
Fixed.